slice_example_test.go

```diff
@@ -479,3 +481,17 @@ func ExampleIsSortedByKey() {
 
 	// Output: true
 }
+
+func ExampleSliceToSet() {
+	list := []string{"a", "b", "d"}
```

Comment:
Can you add more test , in which we have duplicate entries in the slice ?

Reply:
Sure, I fixed it! Please review it again.